examples/linear_model/plot_quantile_regression.py

```diff
@@ -0,0 +1,79 @@
+"""
+==============
+Quantile regression
+==============
+
+Plot the prediction of different conditional quantiles.
+
+The left figure shows the case when error distribution is normal,
+but variance is not constant.
+
+The right figure shows example of an asymmetric error distribution
+(namely, Pareto).
+
+The second part of the code shows that LinearRegression minimizes RMSE,
+while QuantileRegressor minimizes MAE, and both do their own job well.
```

Comment:
Move this paragraph before the actual section of the code when it is computed. Use sphinx-gallery formatting there:
```
#########################################################################
#
# The second part of the code shows that LinearRegression minimizes RMSE,
# while QuantileRegressor minimizes MAE, and both do their own job well.
```
(except also keep the lines <80 characters long).